internal/action/tab.go

```diff
@@ -399,3 +434,14 @@ func (t *Tab) CurPane() *BufPane {
 	}
 	return p
 }
+
+// MoveTab moves the specified tab to the given index
+func (tl *TabList) MoveTab(t *Tab, i int) {
```

Comment:
It's better to put this method not at the very end of `tab.go` but together with other `TabList` methods (for example, after `AddTab()` or `RemoveTab()`)?